requirements/base.txt

```diff
@@ -55,7 +55,7 @@ chardet==3.0.4
 coloredlogs==14.0
 constantly==15.1.0
 croniter==0.3.36
-daphne==3.0.1
+daphne==2.5.0
```

Comment:
This upgrade was breaking locally for me:

```
HTTP GET / 500 [0.00, 127.0.0.1:60739]
Traceback (most recent call last):
  File "/Users/jonny/Sites/.venvs/metadeploy/lib/python3.8/site-packages/daphne/http_protocol.py", line 163, in process
    self.application_queue = yield maybeDeferred(
TypeError: __call__() got an unexpected keyword argument 'receive'
```

```
ERROR: Cannot install -r /app/requirements/base.txt (line 9) and daphne==3.0.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested daphne==3.0.1
    channels 2.4.0 depends on daphne~=2.3

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

Reply:
@jgerigmeyer thanks, you're presumably seeing this while we did not because you have the new version of pip with its stricter resolver. Holding it back sounds like the right solution until we take a closer look.